### DISCUSSION TITLE
gpg is disabled; why error?

### DISCUSSION THREAD

**Discussion by trippap** [community/community#37283] [Other Feature Feedback, Questions, & Ideas]
GPG is disabled on github and globally on my computer, yet whenever I try to commit changes from my local repo to github, I get a "git gpg failed to sign the data" error. I've used git config --global commit.gpgsign false to make sure gpg is disabled on my local computer and repos, yet it still tries to use it. How can I get gpg stilled?

**Comment by ossanna16** [comment]
Hi there @trippap and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
